vassal-doc/src/main/readme-referencemanual/ReferenceManual/GlobalOptions.adoc

```diff
@@ -12,11 +12,11 @@ If an option has a _Use Preferences Setting_ choice, selecting it will add an en
 
 [width="100%",cols="50%a,^50%a",]
 |===
-|*Allow non-owners to unmask pieces:*::  By default, only the player who originally masked a piece (see the Mask trait for Game Pieces) is allowed to unmask it.
-This option allows other players to unmask a masked piece.
+|
+*Description:*::  A short description of this component that appears on the main editor dialog (optional).
 
-*Center on opponent's moves:*::  This option will center a map window on an opponent's move when reading a logfile or receiving a move on the server.
-_It is recommended that in most cases this be set to "Use Preferences" so that players can turn this behavior on and off themselves._
+*Allow non-owners to unmask pieces:*::  By default, only the player who originally masked a piece (see the Mask trait for Game Pieces) is allowed to unmask it.
```

Comment:
Would it be useful to actually provide a link to the Mask documentation page here, instead of just referring to it in text?

Reply:
Done